eng/Packages.Data.props

```diff
@@ -391,6 +391,13 @@
     <PackageReference Update="Microsoft.Extensions.Logging.Configuration" Version="2.1.1" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.AI.Inference.Tests'))">
```

Comment:
There's no justification for isolating this to just the AI.Inference library.   We'll either want to approve as a general dependency for tests/samples or not.

@KrzysztofCwalina, @tg-msft :  I'd appreciate your thoughts.

Reply:
added new otel dependencies to general test section (which contained some of them)